src/main/java/org/mskcc/cbio/oncokb/web/rest/CronJobController.java

```diff
@@ -182,6 +205,112 @@ public void checkTrialAccounts() {
         mailService.sendTrialAccountExpiresMail(DAYS_TO_CHECK, userDTOS);
     }
 
+    @GetMapping(path = "/check-exposed-tokens")
+    public void exposedTokensInfo() throws IOException, InterruptedException {
+        List<Token> tokens = tokenService
+                                .findAll()
+                                .stream()
+                                .filter(token -> token.getExpiration().isAfter(Instant.now()))
+                                .collect(Collectors.toList());
+        log.info("Searching exposed tokens pipeline begins...");
+        List<ExposedToken> results = new ArrayList<>();
+        for (Token token : tokens){
+            String q = token.getToken().toString();
+
+            int githubCount = 0;
+            try {
+                GitHub github = new GitHubBuilder().withOAuthToken(applicationProperties.getGithubToken()).build();
+                List<GHContent> gitRes = github.searchContent().q(q).list().toList();
+                githubCount = gitRes.size();
+                if (githubCount == 1 && gitRes.get(0).getHtmlUrl().indexOf("https://github.com/knowledgesystems/oncokb-deployment/") == 0){
```

Comment:
What's this? Why you specify the oncokb-deployment?

Reply:
Because there are some tokens in `/knowledgesystems/oncokb-deployment/` configmaps. If I used my personal github token to check, it will find oncokb tokens in that repo exposed since my github account has the permission to visit this private repo. If our dev github account doesn't have the permission to visit such private repo, then we don't need this.

Reply:
@Leowisd let's remove this clause. oncokbdev account only has the access to the oncokb organization which only contains public repo.